lib/src/lifecycle_module.dart

```diff
@@ -475,7 +547,12 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
         .then(transition.complete)
         .catchError((error, trace) {
       transition.completeError(error, trace);
+      _activeSpan?.setTag('error', true);
```

Comment:
Is there any value to attaching the stack trace to the span as well?

Reply:
There might be although I'd imagine the logging infrastructure is better for the job, and we're already logging these errors.